packages/node_modules/@webex/plugin-authorization-browser/src/authorization.js

```diff
@@ -209,7 +209,8 @@ const Authorization = WebexPlugin.extend({
         this.webex.credentials.set({
           supertoken: token
         });
-      });
+      })
+      .then(() => this.webex.internal.services.initServiceCatalogs());
```

Comment:
@andrewholsted is this what you were talking about? should this get called in plugin-authorization-browser-first-party as well? i believe first-party would be the web-client